data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1453 @@ mission "Stone of our Fathers 5"
+			choice
+				`	(Return the mourning Skadenga to Windblain.)`
+					goto home
+				`	(Leave them here.)`
+			`	You let them know that you only agreed to take them here, not bring them back. You shut your hatch and leave them to fend for themselves.`
+				decline
+			label home
+			`	They thank you and board your ship.`
+				accept
+	on complete
+		event "skadenga mourners" 90
+		dialog
+			`The Skadenga are gladdened to be home. Something about being on <planet> lifts their spirits, and they thank you for your help.`
+
+
+
+event "skadenga mourners"
+
+
+
+event "hjlod waits"
+
+
+
+mission "Stones of Skadenga 2"
+	name "Stones of Skadenga"
+	description "Take the mourning Skadenga to <destination>, so that they can enact the rites of their dead."
+	source "Windblain"
+	destination "Nifel"
+	passengers 8
+	landing
+	to offer
+		has "event: skadenga mourners"
+		has "Stones of Skadenga 1: done"
+	on offer
+		dialog
+			`A group of eight Skadenga approach you. They ask if you'd be willing to take them to <destination> so that they can return their loved-one's stones to their Family-Cairns, offering <payment> for their transportation to.`
+	on complete
+		payment 80000
+
+
+
+mission "Stones of Skadenga 3"
+	name "Stones of Skadenga"
+	description "Return the Skadenga mourners to <destination>."
+	source "Nifel"
+	destination "Windblain"
+	passengers 8
+	to offer
+		has "Stones of Skadenga 2: done"
+	on offer
+		conversation
+			`The Skadenga wander past the ruins of Udainsakr and make their way to the Family-Cairns before the ice-covered Goddess-Stone of Skade. They say their words, shed their tears, and leave their stones.`
+			`	The mourners ask if you can take them back to <planet>. They have already paid you all their money, and cannot afford any  more for a ride home.`
+			choice
+				`	(Return the mourning Skadenga to <planet>.)`
+					goto home
+				`	(Leave them here.)`
+			`	You let them know that you only agreed to take them here, not bring them back. You shut your hatch and leave them to fend for themselves.`
+				decline
+			label home
+			`	They thank you and board your ship.`
+				accept
+	on complete
+		dialog
+			`The Skadenga are happy to be home. The churning seas of <planet> lift their spirits, and they head out into the sea-sweet air.`
+
+
+
+mission "Stones of Skadenga 4"
+	invisible
+	source "Windblain"
+	to offer
+		has "Stones of Skadenga 3: done"
+	on offer
+		conversation
+			`You're surprised to see the Ondurdis walking toward you.`
+			`	She nods politely before speaking. "<first>, I just wanted to thank you for ferrying our Stone-Bearers to <planet> and back. I have added this task to the job board so you can more easily assist in the future. I've also let them know they need enough money to pay for a return trip." The Ondurdis touches your shoulder, and then walks away.`
+				decline
+
+
+
+mission "Stone-Bearers to Nifel"
```

Comment:
This could also just be combined with the mission below into one stopover mission. I don't think it makes sense for a variable amount of passengers to go just one way anyways, unless other captains are taking them too.

Reply:
I have combined it with the other mission, as you requested.